source/websocket.c

```diff
@@ -989,13 +989,13 @@ static void s_shutdown_channel_task(struct aws_channel_task *task, void *arg, en
 
     s_unlock_synced_data(websocket);
     /* END CRITICAL SECTION */
-    websocket->thread_data.is_reading_stopped = true;
-    websocket->thread_data.is_writing_stopped = true;
```

Comment:
Oh man, I totally didn't notice these changes in that other PR.

I'll take over this PR. I'm oncall